package.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
Overall, really nice work filling in the whole Package.json. Glad to see that you populated everything!